pkg/kubectl/resource_printer.go

```diff
@@ -1186,26 +1188,37 @@ func printPersistentVolumeClaim(pvc *api.PersistentVolumeClaim, w io.Writer, wit
 	return err
 }
 
-func printPersistentVolumeClaimList(list *api.PersistentVolumeClaimList, w io.Writer, withNamespace bool, wide bool, showAll bool, columnLabels []string) error {
+func printPersistentVolumeClaimList(list *api.PersistentVolumeClaimList, w io.Writer, withNamespace bool, wide bool, showAll bool, watch bool, columnLabels []string) error {
 	for _, psd := range list.Items {
-		if err := printPersistentVolumeClaim(&psd, w, withNamespace, wide, showAll, columnLabels); err != nil {
+		if err := printPersistentVolumeClaim(&psd, w, withNamespace, wide, showAll, watch, columnLabels); err != nil {
 			return err
 		}
 	}
 	return nil
 }
 
-func printEvent(event *api.Event, w io.Writer, withNamespace bool, wide bool, showAll bool, columnLabels []string) error {
+func printEvent(event *api.Event, w io.Writer, withNamespace bool, wide bool, showAll bool, watch bool, columnLabels []string) error {
 	namespace := event.Namespace
 	if withNamespace {
 		if _, err := fmt.Fprintf(w, "%s\t", namespace); err != nil {
 			return err
 		}
 	}
+
+	// While watching event, we should print absolute time.
+	var FirstTimestamp, LastTimestamp string
+	if watch {
```

Comment:
These params could be absolute time or relative time. I think "xxTimestamp" is enough.